### PR TITLE
Post-godmode refactor cleanup

### LINKED ISSUE #86709: Snow bears are god moded, where previously ancient ones only were
### Client Version:

515.1643

### Issue Summary:

Spawn a basic/bear/snow.  Observe it has god mode trait.

Issue sourced here https://github.com/tgstation/tgstation/pull/86596/files#r1761801220 in PR #86596

### Round ID:

[238153](https://scrubby.melonmesa.com/round/238153)

### Test Merge Information:

- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Adds new station map Nebulastation](https://www.github.com/tgstation/tgstation/pull/84826)
- [GetFlatIcon improvements.](https://www.github.com/tgstation/tgstation/pull/86077)

### Reproduction Steps:

See above

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Adds some flags that went missing after #86596
Closes #86709

## Changelog
:cl:
fix: Snow bears are no longer impervious to all sources of damage
fix: Cameras, living floors and ghost of poly no longer can be pushed around
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
refactor in the name =/= a refactor PR

**Comment by MrMelbert** [comment]
note for the CL: `CANPUSH` != `can push things`, `CANPUSH` = `can be pushed`. likewise for all the other flags (`CANSTUN` etc)

**Comment by Ghommie** [comment]
I'm onflicted whether I should speedmerge it or wait the remaining 17 hours or so out.

**Comment by SmArtKar** [comment]
> I'm onflicted on whether I should speedmerge it or wait the remaining 17 hours or so out.

its not really that bad (except icebox oh god oh fuck) because the only map that its relevant on already causes suffering to miners

File: code/modules/mob/living/basic/space_fauna/bear/_bear.dm [hunk 1]
```diff
@@ -90,13 +90,14 @@
...
 
 /mob/living/basic/bear/snow/ancient
 	name = "ancient polar bear"
 	desc = "A grizzled old polar bear, its hide thick enough to make it impervious to almost all weapons."
 	gold_core_spawnable = NO_SPAWN
 
-/mob/living/basic/bear/snow/Initialize(mapload)
+/mob/living/basic/bear/snow/ancient/Initialize(mapload)
 	. = ..()
 	add_traits(list(TRAIT_GODMODE, TRAIT_SNOWSTORM_IMMUNE), INNATE_TRAIT)
```
Comment (ninjanomnom) [inline code/modules/mob/living/basic/space_fauna/bear/_bear.dm:102]: Shouldn't the regular snow bear still be snowstorm immune?

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/basic/space_fauna/bear/_bear.dm (hunk 1 above)
Comment (SmArtKar) [reply code/modules/mob/living/basic/space_fauna/bear/_bear.dm:102]: arrrgh right

**Pull request merged**